ci-operator/templates/openshift/installer/cluster-launch-installer-src.yaml

```diff
@@ -225,8 +236,8 @@ objects:
 
         if [[ "${CLUSTER_TYPE}" == "aws" ]]; then
             cat > /tmp/artifacts/installer/install-config.yaml << EOF
-        apiVersion: v1beta4
-        baseDomain: ${BASE_DOMAIN}
+        apiVersion: v1
```

Comment:
Is v1 supported back to rc5?  Note from now on you can't bump this ever because 4.1 clusters have to be able to install using this template (templates have to work for all 4.y GA versions).

Reply:
We GA'd the v1 API before rc4 https://github.com/openshift/installer/pull/1589 (April 14)